src/persist/src/indexed/mod.rs

```diff
@@ -742,10 +742,12 @@ impl<L: Log, B: Blob> Indexed<L, B> {
             }
         }
 
-        for (id, seal_ts) in seal_updates.iter() {
+        for (id, seal) in seal_updates.iter() {
             if let Some(listen_fns) = self.listeners.get(&id) {
                 for listen_fn in listen_fns.iter() {
-                    listen_fn(ListenEvent::Sealed(*seal_ts));
+                    for seal_ts in seal.elements().iter() {
+                        listen_fn(ListenEvent::Sealed(*seal_ts));
```

Comment:
Hmm I wonder if ListenEvent::Sealed should take an Antichain. Can you put a TODO for us to think about it later?